source/user-manual/reference/ossec-conf/syscheck.rst

```diff
@@ -1,6 +1,6 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 .. meta::
-  :description: The ossec.conf file is the main configuration file on the Wazuh manager and it also plays an important role on the agents. Learn more about it and check out an example here. 
+  :description: The manager.conf file is the main configuration file on the Wazuh manager and it also plays an important role on the agents. Learn more about it and check out an example here. 
```

Comment:
What is this description? Is this necessary?

Reply:
Removed, done.